department-of-veterans-affairs/caseflow
# AMA Upcoming Hearing Days & Daily Docket

Resolves #8474 #8586 

### Description
This PR updates hearing prep's upcoming hearings and daily docket pages to be usable with AMA hearings.

### Testing Plan
1. Create an AMA hearing assigned to BVAAABSHIRE
1. Navigate to upcoming hearing days as BVAAABSHIRE and confirm the legacy and ama hearings are displayed
1. Click through to the daily docket for both an AMA and a legacy hearing day and confirm the daily dockets are displayed and the save functionality works

### Future PR
After this PR is merged, I'm going to submit another PR to address these issues:
1. Hearing times are off on AMA appeals
1. Feature tests
1. All "C"s, "V"s, and "T"s should be replaced by a reference to the request_types constant
1. Additional info in the daily docket header
